ocs_ci/ocs/utils.py

```diff
@@ -1284,3 +1284,45 @@ def thread_init_class(class_init_operations, shutdown):
         if shutdown == 0:
             executor.shutdown(wait=True)
             return
+
+
+def collect_pod_container_rpm_package(dir_name):
+    """
+    Collect information about rpm packages from all containers from
+    Args:
+        dir_name(str): directory to store container rpm package info
+    """
+    from ocs_ci.ocs.resources import pod
```

Comment:
can you please move this import outside the function at the beginning of the file?

Reply:
Since utils.py is imported in pod.py, importing at the beginning causes deadlock. Hence i added import statement within specific function

Reply:
> 
please add the comment for circular dependancy